.gitignore

```diff
@@ -0,0 +1,7 @@
+mykey-pair
+mykey-pair.pub
+.terraform
+.terraform.lock.hcl
+terraform.tfstate
+ignore.TXT
+terraform.tfstate.backup
```

Comment:
Ignora de qualquer pasta, não apenas da raíz

Reply:
Depois de atualizar o .gitignore, e antes do git add, executar esse comando:
`it rm -r --cached . `
Motivo: Arquivo de state ainda está subindo aqui